app/jobs/sms_sender_confirmation_job.rb

```diff
@@ -4,7 +4,7 @@ class SmsSenderConfirmationJob < ActiveJob::Base
   def perform(code, mobile)
     TwilioService.new.send_sms(
       to: mobile,
-      body: "Your #{APP_NAME} phone confirmation code is: #{code}."
+      body: "#{code} is your #{APP_NAME} one-time passcode."
```

Comment:
Will.. if you don't agree and we decide want the messages to be the same then we should also remove this file completely and just use the sms_sender_otp_job.rb for both.